### PR TITLE
news post: results of PSC election + PSC chair change

### PR DESCRIPTION
**Pull request by veroandreo**
Here's the promised news draft. It includes a pic of the PSC virtual meeting.

Both Markus and me wrote something about ourselves given the chair change, but please feel free to remove, shrink, edit. Suggestions are welcome :)

This is how it looks like:

image

### CODE REVIEW

File: content/news/2021_02_05_new_grass_psc.md
Comment (neteler) [inline content/news/2021_02_05_new_grass_psc.md:3]:
```suggestion
date: 2021-02-05T23:12:00+02:00
```

File: content/news/2021_02_05_new_grass_psc.md
Comment (petrasovaa) [inline content/news/2021_02_05_new_grass_psc.md:23]: Add diacritics for consistency

```suggestion
* Anna Petrášová     (88)
* Helena Mitášová    (86)
* Martin Landa          (83)
* Verónica Andreo       (76)
* Moritz Lennert        (74)
* Václav Petráš         (68)
```

**Review by petrasovaa — APPROVED** [review]

**Review by wenzeslaus — APPROVED** [review]

**Comment by neteler** [comment]
Before merging, let's also add the assigned roles (in a beautified way):

PSC roles & tasks (for 3 years)

- Chairperson - Vero
- Treasurer - Moritz
- Release manager - Markus, Martin
- Infrastructure manager - Markus
- Translation manager - Huidae
- Website & Marketing manager - Michael, Vero
- Github manager - Vaclav

**Review by mlennert — APPROVED** [review]
Nice text, thank you !

I agree with Markus that the list of roles within the PSC should have its place here as well.

**Comment by veroandreo** [comment]
I have just added a new section with PSC roles

image

File: content/news/2021_02_05_new_grass_psc.md
Comment (neteler) [inline content/news/2021_02_05_new_grass_psc.md:74]:
```suggestion
In addition to the chair role, in the first [PSC](https://trac.osgeo.org/grass/wiki/PSC) meeting, we have
```

**Review by neteler — APPROVED** [review]

**Comment by neteler** [comment]
We still need approval/changes by @cmbarton, @hmitaso, and @HuidaeCho

**Review by HuidaeCho — APPROVED** [review]

**Review by cmbarton — APPROVED** [review]

**Review by hmitaso — COMMENTED** [review]
It all looks good to me

**Review by hmitaso — APPROVED** [review]
It all looks good to me

**Pull request merged**
